### PR TITLE
fix 4857: show help for set-addrs

### LINKED ISSUE #4857: Show help menu when `lotus-miner actor set-addrs` is called without argument
Most lotus commands print a help screen if no argument is provided. The `lotus-miner actor set-addrs` does not, rather instantly drops a transaction on chain that sets the address to `nil`. I'd say this is probably not what a user wants to do :)

### PR DESCRIPTION
**Pull request by deaswang**
fix #4857

### CODE REVIEW

File: cmd/lotus-storage-miner/actor.go [hunk 1]
```diff
@@ -70,7 +75,10 @@ var actorSetAddrsCmd = &cli.Command{
 		ctx := lcli.ReqContext(cctx)
 
 		var addrs []abi.Multiaddrs
-		for _, a := range cctx.Args().Slice() {
+		for _, a := range args {
```
Comment (arajasek) [inline cmd/lotus-storage-miner/actor.go:78]: Is this to enable a way to unset all addresses? If so, I'd rather do the following:

- Add a `--unset` boolean flag
- If it's supplied with no arguments, unset the miner addresses (what this currently does)
- If it's supplied with arguments, fail 
- If it isn't supplied, and yet no arguments are provided, fail

**Review by arajasek — COMMENTED** [review]
Thanks for taking the initiative! I have one question / suggestion.

File: cmd/lotus-storage-miner/actor.go [hunk 2]
```diff
@@ -53,8 +53,22 @@ var actorSetAddrsCmd = &cli.Command{
...
+		},
 	},
 	Action: func(cctx *cli.Context) error {
+		args := cctx.Args().Slice()
+		unset := cctx.Bool("unset")
+		if len(args) == 0 && !unset {
+			return cli.ShowSubcommandHelp(cctx)
+		}
+		if len(args) > 0 && unset {
+			return fmt.Errorf("unset should with no arguments")
```
Comment (magik6k) [inline cmd/lotus-storage-miner/actor.go:69]:
```suggestion
			return fmt.Errorf("unset can only be used with no arguments")
```

**Review by arajasek — APPROVED** [review]
Looks good with Magik's suggested change

**Review by magik6k — APPROVED** [review]

**Comment by magik6k** [comment]
Thanks!

**Pull request merged**